config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace/hypertrace_api_trace_attributes.conf

```diff
@@ -208,6 +208,15 @@ commands = [
               scope: API_TRACE,
               sources: [QS],
               type: ATTRIBUTE
+            },
+            {
+              fqn: Api.Trace.total_span_count,
+              key: totalSpanCount,
+              value_kind: TYPE_INT64,
+              display_name: Total Span Count,
+              scope: API_TRACE,
+              sources: [QS],
+              type: ATTRIBUTE
```

Comment:
Shouldn't this be `type: METRIC` ?

Reply:
metrics don't exist on records, only on aggregations (entities)

Reply:
Not sure what you mean by a record. But if i want something like the min/max/99perc of total span count across all API traces, how is it different from the metrics on an entity?

Reply:
The type represents the data at a single result row level. So at that single result row  an attribute represents something that returns as a value, while a metric representing something that requires an aggregation operation. For an entity, a single result row is implicitly aggregated from many records (I'm calling a record something like a span or trace that is not, itself implicitly aggregated). So on an entity something like duration is a metric and you would ask for p99 (or some other agg operation) of it. On the other hand, something like name is an attribute - no aggregation is required (or supported). For a record scope (that is, not implicitly aggregated), no aggregations are supported at the row level.

This isn't to be confused by running queries via explorer which support general query semantics on top of these scopes where grouping would require aggregations for either attributes or metrics.